build.gradle

```diff
@@ -55,8 +55,14 @@ repositories {
         }
     }
     else {
-        jcenter()
         mavenCentral()
+        maven {
+            url "https://maven.pkg.github.com/pravega/pravega"
+            credentials {
+                username = "pravega-public"
+                password = "\u0067\u0068\u0070\u005F\u0048\u0034\u0046\u0079\u0047\u005A\u0031\u006B\u0056\u0030\u0051\u0070\u006B\u0079\u0058\u006D\u0035\u0063\u0034\u0055\u0033\u006E\u0032\u0065\u0078\u0039\u0032\u0046\u006E\u0071\u0033\u0053\u0046\u0076\u005A\u0049"
```

Comment:
Is this a plaintext password? Why is this not a security concern?

Reply:
This is how it is specified in Pravega README: https://github.com/pravega/pravega#snapshot-artifacts
I understand that the `pravega-public` user has read-only permissions, just to fetch artifacts.